testdata/validate/escapestdout.log

```diff
@@ -0,0 +1,19 @@
+InputFile: ../escape.crs
+ConfigFile: bde_copy.cfg
+ConfigFile: ../test1.cfg
+OutputFile: -
+TableName: crs_mark
+InputFields: id|nod_id|status|type|category|country|beacon_type|protection_type|maintenance_level|mrk_id_dist|disturbed|disturbed_date|mrk_id_repl|replaced|replaced_date|mark_annotation|wrk_id_created|audit_id|desc
+OutputFields: id|nod_id|status|type|category|country|beacon_type|protection_type|maintenance_level|mrk_id_dist|disturbed|disturbed_date|mrk_id_repl|replaced|replaced_date|mark_annotation|wrk_id_created|audit_id|desc
+BdeStartTime: 2010-04-03 21:25:43
+BdeEndTime: 2010-05-01 21:14:42
+BdeSize: 1432
+
+Warning:  Invalid pipe character in field in field desc
+At record 1 of ../escape.crs
+36758676|36758676|COMM|OTHR||NEWZ|NB||||N|||N||||957605|||
```

Comment:
this is still mixed output and logfile ?

Reply:
I updated the test file names, this is escape2.log now, but isn't mixed only the logfile is written to it, the output file is written to escape2.out.